R/extract_diagnostics.R

```diff
@@ -0,0 +1,161 @@
+#' Extract diagnostic metrics from model fit and data
+#'
+#' This function extracts various diagnostic metrics from a fitted `epinow2`
+#' model and provided data. It checks for low case counts and computes
+#' diagnostics from the fitted model, including the mean acceptance
+#' statistic, divergent transitions, maximum tree depth, and Rhat values.
+#' These diagnostics are then flagged if they exceed specific thresholds,
+#' and the results are returned as a data frame.
+#'
+#' @param fit A list containing the model fit object from `epinow2`, which
```

Comment:
Does the list contain the model fit object or the list is the model fit object? I'd assume the later but what you've written sounds like the former

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/41/commits/88b46800dbc647a5de78112c3f4e67b8a9218cce